src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Printable {
+    private String color;
```

Comment:
add setter

Reply:
If I never used setter in my program why I would add it? 

In that abstract class I used constructor and just aded getter because of in future printing I need it when I @Overrided draw(). I got color and add to "sout".

Reply:
> add setter

If I never used setter in my program why I would add it?

In that abstract class I used constructor and just aded getter because of in future printing I need it when I @Overrided draw(). I got color and add to "sout".